src/adapters/lokijs/worker/encodeQuery/index.js

```diff
@@ -86,6 +87,18 @@ const noNullComparisons: OperatorFunction => OperatorFunction = operator => valu
   $and: [operator(value), weakNotEqual(null)],
 })
 
+const like: OperatorFunction = value => {
+  if (typeof value === 'string') {
+    return {
+      $regex: new RegExp(value.replace(/%/g, '.*').replace('_', '.'), 'i'),
```

Comment:
Shouldn't second replace also use regex? `/_/g, '.'` Without you would only replace first occurrence of `_` I think.